knowledge-portal/source/README.md

```diff
@@ -0,0 +1,36 @@
+# Usage
+
+## Adding this pipeline to your repo
+
+You will have to modify the existing `gitlab-ci.yml` file to include:
+
+```yaml
+    stages:
+        - copy_to central_repo
+    include:
+        - remote: 'https://github.com/detecttechnologies/Gitlab-CI-CD-Templates/raw/main/knowledge-portal/source/.gitlab-ci.yml'
+
+```
+## Configure CI/CD variables
+
+1. Go to `Settings >> CI/CD >> Variables`
+2. Click on `Add variable`. 
+3. Give the name `PUSH_TOKEN` in `Key` field and `Value` field will be provided by Platform Team.
+4. Click on `Add variable` button to confirm changes. 
+
+## Creating `manifest.txt` file
```

Comment:
Rather than calling our file just `manifest.txt`, can we call it `docs-manifest.txt` or something along those lines?
`manifest.txt` is likely to be used by some frameworks/languages that it might create a conflict with.